plugins/catalog-react/api-report.md

```diff
@@ -131,6 +133,23 @@ export type DefaultEntityFilters = {
   text?: EntityTextFilter;
 };
 
+// Warning: (ae-missing-release-tag) "DefaultStarredEntitiesApi" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
+//
+// @public (undocumented)
+export class DefaultStarredEntitiesApi implements StarredEntitiesApi {
+  constructor(opts: { storageApi: StorageApi });
+  // (undocumented)
+  isStarred(entity: Entity): boolean;
+  // (undocumented)
+  star(entity: Entity): Promise<void>;
+  // (undocumented)
+  starredEntities$(): Observable<StarredEntitiesApiObservable>;
```

Comment:
think this should be `starredEntitie$` as the `$` replaces the plural `s`.

`error$` `change$` `log$` etc.